CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.4 (2018-11-30)
+
+- Fixed `SkyListBuilderViewGridsModule` exports to include required submodules and `SkyGridModule`. [#9](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/9)
```

Comment:
We should update the notation here to be consistent with the SKYUX changelog:
`[blackbaud/skyux-list-builder-view-grids#9](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/9)`

Reply:
Do we need that notation here? The way I was interpreting the SKY UX changelog is that we need that notation when the link points to a PR in a different repo, but not when it points to a notation within the same repo as the changelog. Since this link points to a PR in the same repo as the changelog, I don't think we need this.

Reply:
I would agree with @blackbaud-johnly here @Blackbaud-AlexKingman

Reply:
@blackbaud-conorwright - I know you had already made the change but FYI

Reply:
done

Reply:
reverted

Reply:
Yep, he's right. Didn't double check which repo I was in. Good catch!